client/src/app/worker/autoupdate-stream.ts

```diff
@@ -291,7 +291,7 @@ export class AutoupdateStream {
 
             return decompressedString;
         } catch (e) {
-            console.warn(`Received uncompressed message from autoupdate.`);
+            console.warn(`Received uncompressed message from autoupdate.`, content);
```

Comment:
When receiving an uncompressed message it can be looked up in the network tab. For that reason I think this does not need to be output here.

```suggestion
            console.warn(`Received uncompressed message from autoupdate.`);
```